docs/features/image-name-substitution.md

```diff
@@ -0,0 +1,9 @@
+# Image name substitution
```

Comment:
To add a new page you'll need to add it here: https://github.com/testcontainers/testcontainers-node/blob/cce5bccd7f4dee04de3948bdf7dae52e21969883/mkdocs.yml#L37-L43. However I wonder if this would be better off in the `Images` feature page?

Reply:
Sounds good, I'll add it to `Images`.